Restore koa params method to contain the matched concord method

vogre opened the pull request:
Lost in https://github.com/binaris/concord/commit/8de0aff9f33c0bf6696c71ae8221d5c0384f558d

Commit:
Restore koa params method to contain the matched concord method

bergundy commented on templates/ts/server-koa.ts:
```diff
@@ -64,6 +64,7 @@ export class {{name}}Router {
     const extractContext = this.handler.extractContext.bind(this.handler);
     {{/serverOnlyContext}}
 
+    this.koaRouter.post('/:method', (_ctx, next) => next());
```
Isn't there a better way to get this?

bergundy replied:
`ctx.state.method`?

Commit:
CR: Create Koa app directly instead of exteding TestServer

Commit:
CR: Use koa state for passing concord method

Breaking away with backwards compatibility is safe since old versions
are still available.
